gen/MotController_data.yml

```diff
@@ -1125,7 +1128,48 @@ c_MotController_ConfigGetCustomParam:
     :returns: Value of the custom param.
 c_MotController_ConfigSetParameter:
   code: |
-    raise NotImplementedError
+    if param == ParamEnum.eOpenloopRamp:
+        self.hal_data['open_loop_ramp'] = value
+    elif param == ParamEnum.eClosedloopRamp:
+        self.hal_data['closed_loop_ramp'] = value
+    elif param == ParamEnum.eNeutralDeadband:
+        self.hal_data['neutral_deadband'] = value
+    elif param == ParamEnum.ePeakPosOutput:
+        self.hal_data['peak_fwd_output'] = value
+    elif param == ParamEnum.eNominalPosOutput:
+        self.hal_data['nom_fwd_output'] = value
+    elif param == ParamEnum.ePeakNegOutput:
+        self.hal_data['peak_rev_output'] = value
+    elif param == ParamEnum.eNominalNegOutput:
+        self.hal_data['nom_rev_output'] = value
```

Comment:
I suspect the PID param values sent are different to what the config API accepts.

Reply:
… and this mobile client is broken and attached these comments to the wrong line >_>